dev-docs/modules/wurflRtdProvider.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page_v2
+title: WURFL RTD Provider Module
+display_name: WURFL RTD Module
+description: WURFL Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : wurflRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+{: .alert.alert-warning :}
+This module loads a dynamically generated JavaScript from prebid.wurflcloud.com
+
+# WURFL RTD Submodule
+
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Description
+
+The WURFL RTD module enriches the OpenRTB 2.0 device data with [WURFL data](https://www.scientiamobile.com/wurfl-js-business-edition-at-the-intersection-of-javascript-and-enterprise/).
+The module sets the WURFL data in `device.ext.wurfl` and all the bidder adapters will always receive the low entry capabilites like `is_mobile`, `complete_device_name` and `form_factor`. 
+
+For a more detailed analysis bidders can subscribe to detect iPhone and iPad models and receive additional [WURFL device capabilities](https://www.scientiamobile.com/capabilities/?products%5B%5D=wurfl-js).
+
+## User-Agent Client Hints
+
+WURFL.js is fully compatible with Chromium's User-Agent Client Hints (UA-CH) initiative. If User-Agent Client Hints are absent in the HTTP headers that WURFL.js receives, the service will automatically fall back to using the User-Agent Client Hints' JS API to fetch [high entropy client hint values](https://wicg.github.io/ua-client-hints/#getHighEntropyValues) from the client device. However, we recommend that you explicitly opt-in/advertise support for User-Agent Client Hints on your website and delegate them to the WURFL.js service for the fastest detection experience. Our documentation regarding implementing User-Agent Client Hint support [is available here](https://docs.scientiamobile.com/guides/implementing-useragent-clienthints). 
+
+## Usage
+
+### Build
+
+```bash
+gulp build --modules="rtdModule,wurflRtdProvider,appnexusBidAdapter,..."  
```

Comment:
Is this true? I thought that RTD submodules could be set up to automatically pull in the base RTD module. @dgirardi ?

Reply:
if the module is listed [here](https://github.com/prebid/Prebid.js/blob/27a2d3b19d0fbc01b37a784bcf316831e97e1f12/modules/.submodules.json#L59) it automatically pulls in the base RTD module, and listing it explicitly is unnecessary.

Reply:
Should be https://github.com/prebid/Prebid.js/pull/11840 updated to list `wurflRtdProvider` in the `.submodmodule.json` file?

Reply:
Yes - that would enable you to drop adding 'rtdModule' in the build command.

Reply:
Updated in 6354b8d and https://github.com/prebid/Prebid.js/pull/11840/commits/b2a8e2f380277b998cc55ec9ff53ccd1bccb87ff, thanks for pointing out.